public/robots.txt

```diff
@@ -1,5 +1,5 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 #
 # To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+User-agent: *
```

Comment:
Basically, you can target certain directives to specific crawlers (which identify by their User-agent) or just define rules that apply to all crawlers.